com.dxx_redux.D1X-Redux.json

```diff
@@ -0,0 +1,88 @@
+{
+  "id": "com.dxx_redux.D1X-Redux",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "d1x-redux",
+  "rename-desktop-file": "d1x-redux.desktop",
+  "rename-icon": "d1x-redux",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--share=ipc",
+    "--device=all"
+  ],
+  "modules": [
+    "shared-modules/SDL/sdl12-compat.json",
+    "shared-modules/linux-audio/fluidsynth2.json",
+    "SDL_mixer-git.json",
+    "shared-modules/glu/glu-9.json",
+    "shared-modules/glew/glew.json",
+    "shared-modules/physfs/physfs.json",
+    {
+      "name": "D1X-Redux",
+      "buildsystem": "cmake-ninja",
+      "subdir": "d1",
+      "config-opts": [
+          "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+          "-DXDGCONFIG=1"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/dxx-redux/dxx-redux/archive/refs/tags/v1.0.tar.gz",
+          "sha256": "500d70a1c6e3391317b12b7714c2539fb31438e99d32d6016013026749e26bea",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 1,
+            "url-template": "https://github.com/dxx-redux/dxx-redux/archive/refs/tags/v$version.tar.gz"
+          }
+        },
+        {
+          "type": "patch",
+          "path": "flatpak-cmake.patch"
+        },
+        {
+          "type": "patch",
+          "path": "flatpak-soundfont.patch"
+        },
+        {
+          "type": "patch",
+          "path": "xdgconfig.patch"
+        }
+      ]
+    },
+    {
+      "name": "TimGM6mb",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/share/sounds/sf3",
+        "install -m644 TimGM6mb.sf2 /app/share/sounds/sf3/default-GM.sf3"
```

Comment:
```suggestion
        "install -Dm644 TimGM6mb.sf2 /app/share/sounds/sf3/default-GM.sf3"
```

bit is an SF2...

Reply:
I'll merged mkdir and install.
default-GM.sf3 is the only default filename for fluidsynth, this is also where debian links TimGM6mb to (https://salsa.debian.org/multimedia-team/timgm6mb-soundfont/-/blob/master/debian/timgm6mb-soundfont.postinst).